composer.json

```diff
@@ -16,8 +16,8 @@
         }
     ],
     "require": {
-        "php": "^8.1",
-        "illuminate/support": "^10.0"
+        "php": "^8.1|^8.2|^8.3",
```

Comment:
This isn't necessary. `^8.1` will already include 8.1, 8.2, 8.3, and 8.4 (once it's out in a couple of weeks).

Reply:
you're right!, I updated the PR, thanks!